### PR TITLE
just: add page

### PR DESCRIPTION
**Pull request by boldandbrad**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 1.2.0

### CODE REVIEW

File: pages/common/just.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Run a recipe specified in the justfile:
+
+`just {{recipe}}`
+
+- Initialize new justfile in project root:
+
+`just --init`
+
+- Edit justfile with editor given by $VISUAL or $EDITOR, falling back to vim:
```
Comment (navarroaxel) [inline pages/common/just.md:14]: We used to use this phrase
```suggestion
- Edit justfile in the default editor:
```
Is the `justfile` is the current directory?

File: pages/common/just.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# just
+
+> A handy way to save and run project-specific commands.
```
Comment (navarroaxel) [inline pages/common/just.md:3]: We try to use a objective command description
```suggestion
> Save and run project-specific commands.
```

File: pages/common/just.md (hunk 1 above)
Comment (boldandbrad) [reply pages/common/just.md:14]: just is smart enough to find justfile in the current project root (I think it uses git)

Comment (boldandbrad) [reply pages/common/just.md:14]: otherwise current directory, yes

**Review by einverne — APPROVED** [review]
Looks good.

**Review by navarroaxel — APPROVED** [review]

**Comment by boldandbrad** [comment]
Thank you!

**Comment by CleanMachine1** [comment]
Thank you, for your contribution :smile:

**Pull request merged**